samples/cellular/nrf_cloud_mqtt_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -0,0 +1,125 @@
+# Note that this overlay is not compatible with nrf91! Use something like the nrf53 as the host
+# microcontroller.
+
+## Disable Modem/LTE/nrf91-specific features
```

Comment:
I'm using `##` as section start, and `#` as notes-within-section -- It's intended as a readability thing. I think I saw someone else doing something similar, which is where I got the idea, although I've since forgotten where I saw it